config/tools.json

```diff
@@ -0,0 +1 @@
+{"generator":[{"title":"Sample Tool","description":"Tool for testing","links":{"websiteUrl":"https://akshatnema.netlify.app","docsUrl":"","iconUrl":"","repoUrl":"https://github.com/akshatnema/Login-Registration-project/blob/61855e7365a881e98c2fe667a658a0005753d873/.asyncapi-tool"},"filters":{"languages":"javascript","technology":["react"],"categories":["generator"],"hasCommercial":false,"isAsyncAPIOwner":false}}]}
```

Comment:
maybe we can put some real tool here?

Reply:
This is not the sample tool I added. This is extracted final JSON of the set of tools. If you want have a real tool represented here, kindly add the `.asyncapi-tool` file in any of the AsyncAPI tools repo and it will be shown here.

Reply:
actually we should add one manually, to also make sure that it is still possible and that we do not regenerate the file and lost tools that were added manually

how about https://gitlab.com/djencks/asyncapi-asciidoc-template ?

Reply:
Ok, this is what we missed in the last meeting to discuss. :sweat_smile:.

So, my idea is to have this `tools.json` file regenerated everytime the github workflow works and for grabbing the tools added manually, we can have a seperate markdown file to add the details of tools in tools.json file using some computation (as we did for GitHub). And if you want to have a real tool scenario added here, why not you suggest me to add here https://github.com/akshatnema/Login-Registration-project for testing? because if you look into the `.asyncapi-tool` file closely, we are still not finalised with the schema of the file and we are changing it constantly. If you want to have a live demo for a tool in the UI along with scripts running, you can just tell me what correct attributes you want to have for a certain tool and I will add this in my repo for testing.

Reply:
> https://github.com/akshatnema/Login-Registration-project/blob/master/.asyncapi-tool

this is definitely a good example for testing script

and the GitLab one will be perfect for "manual" tools list.

I suggest we do not play with markdown as tools are added in 99% of cases by developers, I bet they know how to use JSON 😄 

so I like idea with 2 separate files. Maybe just:
- `tools_manual.json` for manual additions
- `tools_automated.json` for script weekly updates
- `tools.json` that is a merge of above and used as source for the website